prov/rxd/src/rxd_cq.c

```diff
@@ -633,13 +668,18 @@ static struct rxd_x_entry *rxd_rx_atomic_fetch(struct rxd_ep *ep,
 			struct rxd_atom_hdr *atom_hdr)
 {
 	struct rxd_x_entry *rx_entry;
+    struct rxd_peer *peer_entry;
 	int ret;
 
 	rx_entry = rxd_get_rx_entry(ep, base_hdr->type);
+    
 	if (!rx_entry) {
-		FI_WARN(&rxd_prov, FI_LOG_EP_CTRL, "could not get tx entry\n");
+		FI_WARN(&rxd_prov, FI_LOG_EP_CTRL, "could not get rx entry\n");
```

Comment:
Separate this into a separate typo patch

Reply:
reverted for now will create a patch for this.